src/Repository/TeamsRepository.php

```diff
@@ -19,6 +20,51 @@ public function __construct(ManagerRegistry $registry)
         parent::__construct($registry, Teams::class);
     }
 
+    /**
+     * @return array
+     */
+    public function getTeams(): array
+    {
+        $query = $this->createQueryBuilder('teams');
+        $query
+            ->select(
+                'teams'
+            );
+        return $query->getQuery()->getArrayResult();
+    }
+
+    /**
+     * @param int $id
+     * @return array
+     */
+
+    public function getTeamById(int $id)
+    {
+        $query = $this->createQueryBuilder('teams');
+        $query
+            ->select('teams')
+            ->where($query->expr()->orX($query->expr()->eq('teams.id', $id)));
+
+        return $query->getQuery()->getArrayResult();
+    }
+
+    /**
+     * @param int $teamId
+     * @return array
+     */
+
+//    public function getProjectsByTeamId(int $teamId): array
+//    {
+//        $qb = $this->createQueryBuilder('team');
+//        $qb
+//            ->select('team.name')
+//            ->innerJoin('team.projects', 'projects')
+//            ->where($qb->expr()->eq('team.id', ':team_id'))
+//            ->setParameter('team_id', $teamId);
+//
+//        return $qb->getQuery()->getArrayResult();
+//    }
+
```

Comment:
if you are commenting something, please remove before you pushing into to the develop or master